src/core/chuck_emit.cpp

```diff
@@ -3636,8 +3636,8 @@ t_CKBOOL emit_engine_emit_exp_primary( Chuck_Emitter * emit, a_Exp_Primary exp )
         }
         else if( exp->var == insert_symbol( "pi" ) )
         {
-            double pi = 3.14159265358979323846;
-            emit->append( new Chuck_Instr_Reg_Push_Imm2( pi ) );
+            double ckPi = 3.14159265358979323846;
+            emit->append( new Chuck_Instr_Reg_Push_Imm2( ckPi ) );
         }
```

Comment:
renamed `pi` due to global namespace pollution in the UE environment on macOS (some system framework defined `pi` already)